packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -104,12 +112,15 @@ class ChatMessage extends UIComponent<ReactProps<ChatMessageProps>, ChatMessageS
     timestamp: customPropTypes.itemShorthand,
     onBlur: PropTypes.func,
     onFocus: PropTypes.func,
+    reactionGroup: customPropTypes.itemShorthand,
```

Comment:
client's voice in me would say that originally taken `reactions` name would work better - although we might want to be absolutely pedantic in terms of matching prop names with component types used as a default value, I would rather expect that these two concepts will be decoupled from each other, in general. These are the main reasons for that:
- `reactions` prop name looks nicer to the client
- in general we might foresee that, _for some reason_, `ReactionsGroup` name that was originally taken for this component will change. Then, if we would still strive for strict correspondence between these two, we would need to change the name of the prop in the API as well - and, actually, it might be not the only place where this change would be necessary

Reply:
Yep, the reason I renamed it is correlated with the next comment you've added... 

- I would have `reactions={{items: [....]}}` which looks even more awkward.. With preserving the name `reactionGroup`, I would feel less bad for having items there... 
- At this moment, it seems more consistent with the other `shorthands`. 

These are the only reasions why I have chosen this name. I may change it, but then we will have on the same component the props: `actionMenu` and `reactions` (inconsistency even in the same component's API). Let me know what do you think...

Additional thought, maybe for some of the components that represent group of other components, like the Menu, ButtonGroup, ReactionGroup etc, we may create special shorthand that will be list and will be applied on the collection shorthand that they represent. This would solve lots of these inconsistency and awkward api...

Reply:
yes, absolutely agree - lets just keep this in mind and defer to follow-up PR the necessary changes for shorthand to be able to process array values. This would solve this problem, as well as the problems that @layershifter has reminded us about here: https://github.com/stardust-ui/react/pull/959/files#r260743449

Reply:
Here is the created issue for reference: https://github.com/stardust-ui/react/issues/984